.github/workflows/ci.yaml

```diff
@@ -30,22 +31,24 @@ jobs:
 
   Windows:
     name: Test on Windows
-    runs-on: windows-2019
+    runs-on: windows-latest
```

Comment:
A nice thing with pinning is that the CI won't randomly start breaking when the next runner is released, but would rather require a PR that changes this value to the next one and then we would see if that breaks there.

Having things run on `latest` can be frustrating if e.g. someone want's to submit a small PR that fixes a bug, but then that small PR is blocked because CI is failing because of change brought by the newer runner...

Reply:
You're right though all the other platforms specified in these two GitHub Actions workflows files use the `latest` versions except for this particular step, should I update all the platform versions to the latest version without the `latest` tag? E.g. `ubuntu-22.04`.

Reply:
I indeed had to pin to windows-2019 recently because -latest broke something that I didn't have time to look into.